terraform/load-balancer/test/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "id" {
+  value = module.load_balancer.id
+}
```

Comment:
Pull out name and resource_group_name as well.

Reply:
Just implemented suggested changes. Resolving conversation. Re-open conversation if you have further suggestions or disagree with the changes proposed and implemented.